spec/models/user_spec.rb

```diff
@@ -0,0 +1,123 @@
+require 'rails_helper'
+
+
+describe User do
+  describe "validating user information" do
+	  it "has a valid factory" do
+	    FactoryGirl.create(:user).should be_valid
```

Comment:
you can create new records with shorter form `create(:user)`

to do so, you need to include this in your `rails_helper.rb`:
https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#rspec